duplicate-post-admin.php

```diff
@@ -1239,15 +1320,17 @@ function duplicate_post_add_bulk_filters() {
 }
 
 /**
- * Adds 'Clone' to the bulk action dropdown.
+ * Adds 'Clone' and 'Rewrite & Republish' to the bulk action dropdown.
  *
  * @ignore
  *
  * @param array $bulk_actions The bulk actions array.
- * @return array.
+ * @return array The bulk actions array.
  */
 function duplicate_post_register_bulk_action( $bulk_actions ) {
 	$bulk_actions['duplicate_post_clone'] = esc_html__( 'Clone', 'duplicate-post' );
+	$bulk_actions['duplicate_post_rewrite_republish'] = esc_html__( 'Rewrite & Republish', 'duplicate-post' );
```

Comment:
PHPCS fails locally for me (Equals sign not aligned...). Not sure why it passes on Travis. @jcomack any thoughts?

Reply:
Clarified with Jimmy this actually throws a warning (not an error) so Travis passes and this is the expected behavior. Also, for now there's no limit set to the warnings so the general CS approach should be addressed later.